app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+android:id="@+id/activity_main"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+android:gravity="center"
+android:orientation="vertical">
+
+
+    <TextView
+        android:id="@+id/parking_size_label"
```

Comment:
Please review that all your views IDs comply with [Mobile Resources Style for Android](https://docs.google.com/document/d/1eImvh7-1tTKn6e2ivGDChqoD76nw4VzNk_ompvz4kJg/edit#heading=h.5r09p6gu0g3o)